### PR TITLE
Documentation: add information about existing documentation and APIs

### PR DESCRIPTION
**Pull request by anslpa**

### CODE REVIEW

**Review by cbellot000 — APPROVED** [review]

File: docs/source/concepts/waysofusing.rst [hunk 1]
```diff
@@ -1,26 +1,82 @@
...
+
+DPF as a standalone application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Standalone DPF** can be used through any of the previously mentioned APIs.
 
 CPython
 -------
-Standalone DPF uses CPython and can be accessed with any Python console. 
+Especially, DPF Standalone capabilities can be enabled using **CPython** and can be accessed via any Python console.
```
Comment (PProfizi) [inline docs/source/concepts/waysofusing.rst:69]:
```suggestion
DPF Standalone capabilities can be enabled using **CPython** and can be accessed via any Python console.
```

File: docs/source/concepts/waysofusing.rst [hunk 2]
```diff
@@ -1,26 +1,80 @@
...
+---------------------------------------------------------
+
+DPF application: kernel and operator's libraries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+DPF is a framework that provides data computation capabilities. These capabilities are provided
+through libraries of operators. To learn more about the computed data and the operator concepts, see :ref:`user_guide_concepts`.
+
+A DPF application is always composed of a kernel (DataProcessingCore and DPFClientAPI binaries),
+that enables capabilities by loading libraries of operators (for example mapdlOperatorsCore library
```
Comment (JennaPaikowsky) [inline docs/source/concepts/waysofusing.rst:17]:
```suggestion
that enables capabilities by loading libraries of operators (for example, mapdlOperatorsCore library
```

File: docs/source/concepts/waysofusing.rst [hunk 3]
```diff
@@ -1,26 +1,80 @@
...
+is basic library enabled by DPF).
+**A such application is also called a "DPF Server application"**.
```
Comment (JennaPaikowsky) [inline docs/source/concepts/waysofusing.rst:19]:
```suggestion
This application is also called a **DPF Server application**.
```

File: docs/source/concepts/waysofusing.rst [hunk 4]
```diff
@@ -1,26 +1,80 @@
...
+
+Starting a DPF application, it is possible to customize the list of operator's libraries that the kernel loads.
```
Comment (JennaPaikowsky) [inline docs/source/concepts/waysofusing.rst:21]:
```suggestion
When starting a DPF application, you can customize the list of operator's libraries that the kernel loads.
```

File: docs/source/concepts/waysofusing.rst [hunk 5]
```diff
@@ -1,26 +1,80 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+DPF is a framework that provides data computation capabilities. These capabilities are
+enabled using the DPF Server application.
+These capabilities can be accessed through "client APIs", as shown on the following picture:
+
+.. image:: ../images/drawings/apis_2.png
+
+
+1. DPF server application can be accessed using Ansys Inc product, or
```
Comment (JennaPaikowsky) [inline docs/source/concepts/waysofusing.rst:34]:
```suggestion
1. The DPF server application can be accessed using Ansys Inc product, or
```

File: docs/source/concepts/waysofusing.rst [hunk 6]
```diff
@@ -1,26 +1,80 @@
...
+DPF Server package (see :ref:`ref_getting_started_with_dpf_server`)
+available on the Customer portal
```
Comment (JennaPaikowsky) [inline docs/source/concepts/waysofusing.rst:36]:
```suggestion
available on the Customer portal.
```

File: docs/source/concepts/waysofusing.rst [hunk 7]
```diff
@@ -1,26 +1,80 @@
...
+
+2. Several client APIs are available (CPython, IronPython, C++, and so on)
```
Comment (JennaPaikowsky) [inline docs/source/concepts/waysofusing.rst:38]:
```suggestion
2. Several client APIs are available (CPython, IronPython, C++, and so on).
```

File: docs/source/concepts/waysofusing.rst [hunk 8]
```diff
@@ -1,26 +1,80 @@
...
+
+3. Communication in the same process, or through gRPC (allows to have the client and the servers on different machines)
```
Comment (JennaPaikowsky) [inline docs/source/concepts/waysofusing.rst:40]:
```suggestion
3. Communication in the same process, or through gRPC, allows you to have the client and the servers on different machines.
```

File: docs/source/concepts/waysofusing.rst [hunk 9]
```diff
@@ -1,26 +1,80 @@
...
+
+The **list of available operators when using DPF is independent from the language or API which is used**, it only depends
+on how the DPF application has been initialized.
+
+Most of the DPF capabilities can be accessed using the operators. For more information about the existing operators, see **Operators** tab.
+
+Enhance DPF capabilities
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+*The available DPF capabilities loaded in a DPF application can be enhanced** by creating new operator's libraries.
```
Comment (JennaPaikowsky) [inline docs/source/concepts/waysofusing.rst:52]:
```suggestion
The available DPF capabilities loaded in a DPF application can be enhanced by creating new operator's libraries.
```

File: docs/source/getting_started/compatibility.rst [hunk 10]
```diff
@@ -20,7 +20,14 @@ version 0.3.0 or later. With Ansys 2021 R1, you must use a PyDPF-Core 0.2
 version.
 
 As new features are developed, every attempt is made to ensure backward
-compatibility from the client to the server.
+compatibility from the client to the server. Backward compatibility is ensured for
+the 4 latest Ansys versions. For example, ansys-dpf-core module with 0.8.0 version has been
+developed for Ansys 2023 R2 pre1 release, for 2023 R2 Ansys version. It is compatible with
+2023 R2, 2023 R1, 2022 R2 and 2022 R1 Ansys versions.
+
+**It is strongly encouraged to use the latest packages available**, as far they are compatible
```
Comment (JennaPaikowsky) [inline docs/source/getting_started/compatibility.rst:28]:
```suggestion
**Ansys strongly encourages you to use the latest packages available**, as far they are compatible
```

File: docs/source/index.rst [hunk 11]
```diff
@@ -76,6 +76,53 @@ DPF is developed around two core entities:
 Each DPF capability is developed through operators that allow for componentization
 of the framework. Because DPF is plugin-based, new features or formats can be easily added.
 
+Accessing and enriching DPF capabilities
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Most of the DPF capabilities can be accessed using the operators.
+For more information about the existing operators, see **Operators** tab.
```
Comment (JennaPaikowsky) [inline docs/source/index.rst:83]:
```suggestion
For more information about the existing operators, see the **Operators** tab.
```

File: docs/source/index.rst [hunk 12]
```diff
@@ -76,6 +76,53 @@ DPF is developed around two core entities:
...
+
+Following sections are summarizes. For more detailed content, see :ref:`user_guide_waysofusing`.
```
Comment (JennaPaikowsky) [inline docs/source/index.rst:85]:
```suggestion
The following sections are summaries. For more detailed content, see :ref:`user_guide_waysofusing`.
```

File: docs/source/index.rst [hunk 13]
```diff
@@ -76,6 +76,53 @@ DPF is developed around two core entities:
...
+
+**Accessing DPF Server files**
+
+DPF capabilities are accessible having DPF Server files available. Those files can be accessed using:
```
Comment (JennaPaikowsky) [inline docs/source/index.rst:89]:
```suggestion
DPF capabilities are accessible when DPF Server files are available. These files can be accessed using:
```

File: docs/source/index.rst [hunk 14]
```diff
@@ -76,6 +76,53 @@ DPF is developed around two core entities:
...
+
+- the **Ansys installer**. To use it, download the standard installation using your preferred distribution channel,
```
Comment (JennaPaikowsky) [inline docs/source/index.rst:91]:
```suggestion
- The **Ansys installer**. To use it, download the standard installation using your preferred distribution channel,
```

File: docs/source/index.rst [hunk 15]
```diff
@@ -76,6 +76,53 @@ DPF is developed around two core entities:
...
+and install Ansys following the installer instructions. For information on getting a licensed copy of Ansys,
+visit the `Ansys website <https://www.ansys.com/>`_.
+
+- the DPF Server package (see :ref:`ref_getting_started_with_dpf_server`).
```
Comment (JennaPaikowsky) [inline docs/source/index.rst:95]:
```suggestion
- The DPF Server package (see :ref:`ref_getting_started_with_dpf_server`).
```

File: docs/source/index.rst [hunk 16]
```diff
@@ -76,6 +76,53 @@ DPF is developed around two core entities:
...
+It is independent of the Ansys installer.
+
+**Accessing capabilities with scripting**
+
+- C++ documentations:
```
Comment (JennaPaikowsky) [inline docs/source/index.rst:100]:
```suggestion
- C++ documentation:
```

File: docs/source/index.rst [hunk 17]
```diff
@@ -76,6 +76,53 @@ DPF is developed around two core entities:
...
+
+  1. see Data Processing Framework section in `Platform panel <https://ansysapi.ansys.com/account/secured?returnurl=/Views/Secured/main_page.html?lang=en>`_.
```
Comment (JennaPaikowsky) [inline docs/source/index.rst:102]:
```suggestion
  1. The Data Processing Framework section in `Platform panel <https://ansysapi.ansys.com/account/secured?returnurl=/Views/Secured/main_page.html?lang=en>`_.
```

File: docs/source/index.rst [hunk 18]
```diff
@@ -76,6 +76,53 @@ DPF is developed around two core entities:
...
+
+  2. see `Developer Portal <https://developer.ansys.com/product/DPF-Server-Client-Library/index.xhtml>`_
```
Comment (JennaPaikowsky) [inline docs/source/index.rst:104]:
```suggestion
  2. `Developer Portal <https://developer.ansys.com/product/DPF-Server-Client-Library/index.xhtml>`_
```

File: docs/source/index.rst [hunk 19]
```diff
@@ -76,6 +76,53 @@ DPF is developed around two core entities:
...
+
+- CPython modules documentations:
```
Comment (JennaPaikowsky) [inline docs/source/index.rst:106]:
```suggestion
- CPython modules documentation:
```

**Review by JennaPaikowsky — APPROVED** [review]
Just a few things to update

File: docs/source/concepts/waysofusing.rst (hunk 1 above)
Comment (anslpa) [reply docs/source/concepts/waysofusing.rst:67]: I've used "in particular", thanks!

**Comment by anslpa** [comment]
Hi @JennaPaikowsky, thank you for the complete review. I've propagated the changes on the C++ documentation.

**Pull request merged**